keyboards/xelus/wombat/info.json

```diff
@@ -0,0 +1,295 @@
+{
+    "manufacturer": "Xelus",
+    "keyboard_name": "Wombat",
+    "maintainer": "Xelus22",
+    "bootloader": "stm32-dfu",
+    "diode_direction": "COL2ROW",
+    "dynamic_keymap": {
+        "eeprom_max_addr": 32767
+    },
```

Comment:
This shouldn't be needed, should be set to whatever the wear leveling size is.  Eg, this should only be needed if you want/need to have a different size.